lib/package-managers/npm.js

```diff
@@ -8,34 +8,18 @@ exports.NPM = class {
     this.executableName = 'npm';
   }
 
-  install(packages, workingDirectory) {
-    if (!workingDirectory) workingDirectory = process.cwd();
-
+  install(packages = [], workingDirectory = process.cwd()) {
```

Comment:
The reason we did not give workingDirectory a default value is because cli allows to issue the command from a sub folder (for example `your-app-folder/src`).

The cli finds out correct working directory which **may be different from process.cwd()**. So this `install()` method is always called with an explicit workingDirectory internally. You don't need a default value here, and `process.cwd()` could be wrong default.

Reply:
Ok I will remove the default value but this will functionally be different from the previous original implementation where the if statement did the same fallback if no working dir was passed to this method.

Reply:
Sorry ignore my previous message, I forgot how the code works :(